login/login.py

```diff
@@ -0,0 +1,26 @@
+import sqlite3
+import json
+
+def handleLogin(loginCredentials):
+  with sqlite3.connect("./rare.db") as conn:
+    conn.row_factory = sqlite3.Row
+    db_cursor = conn.cursor()
+    db_cursor.execute("""
+    SELECT 
+        u.id
+    FROM user u
+    WHERE u.email = ?    
+    """, (loginCredentials['email'], ))    
+    data = db_cursor.fetchone()
+    
+    userMatches = db_cursor.rowcount
+    response = {}
+
+
+    if data != None:
```

Comment:
Do we need to add a password check here (or alternatively, in the query itself)? As of now it seems that we're only checking if there is a user with the given email, do we wanna verify their password now too?

Reply:
Good point. I just pushed up changes to check the password.